src/toxcore/messenger/packet/typing.rs

```diff
@@ -0,0 +1,76 @@
+/*! Typing struct.
+*/
+
+use nom::le_u8;
+
+use crate::toxcore::binary_io::*;
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[repr(u8)]
```

Comment:
What is it for?

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=f61bd5f33844aecbf719174b567f1458

Reply:
wow

Reply:
done